DependencyInjection/Configuration.php

```diff
@@ -17,8 +17,8 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('alfatraining_cassandra_session_handler');
+        $treeBuilder = new TreeBuilder('alfatraining_cassandra_session_handler');
```

Comment:
This is only compatible with Symfony 5 (https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Config/Definition/Builder/TreeBuilder.php#L30: 4.4 would throw deprecation notices)